kie-wb-tests/pom.xml

```diff
@@ -52,6 +53,60 @@
         <artifactId>kie-wb-tests-gui</artifactId>
         <version>${version.org.kie}</version>
       </dependency>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-drools-wb-webapp</artifactId>
+        <version>${deployable.version}</version>
+        <classifier>${deployable.classifier.swarm}</classifier>
+      </dependency>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-wb-webapp</artifactId>
+        <version>${deployable.version}</version>
+        <classifier>${deployable.classifier.swarm}</classifier>
+      </dependency>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-wb</artifactId>
+        <version>${deployable.version}</version>
+        <classifier>${deployable.classifier.wildfly10}</classifier>
+        <type>war</type>
+      </dependency>
```

Comment:
We are invoking this from QE's test suite and we need to be able to modify the versions to use productized artifacts. Because of this configuration option missing and at the same time wrongly produced artifact versions by the productization we are now not able to run this test suite at all with DR1.